md2conf/api.py

```diff
@@ -185,24 +185,30 @@ def upload_attachment(
         page_id: str,
         attachment_path: Path,
         attachment_name: str,
+        raw_data: Optional[bytes] = None,
         comment: Optional[str] = None,
         *,
         space_key: Optional[str] = None,
         force: bool = False,
     ) -> None:
         content_type = mimetypes.guess_type(attachment_path, strict=True)[0]
 
-        if not attachment_path.is_file():
+        if not raw_data and not attachment_path.is_file():
```

Comment:
`raw_data` and `attachment_path` seem to be exclusive. I would assume the function would be expected to raise an exception when both are present.